atd-vzv/src/views/nav/CrashTypeSelector.js

```diff
@@ -1,4 +1,5 @@
 import React, { useState, useEffect } from "react";
+import { Button, ButtonGroup } from "reactstrap";
 import styled, { css } from "styled-components";
```

Comment:
Looks like this import is no longer used.

<img width="452" alt="Screen Shot 2020-02-28 at 11 31 43 AM" src="https://user-images.githubusercontent.com/37249039/75570990-ec1b8780-5a1d-11ea-8b8e-63225d59a926.png">

Reply:
Thanks! Removed